testsuite/features/build_validation/add_custom_repositories/add_rocky9_repositories.feature

```diff
@@ -0,0 +1,113 @@
+    And I select "RHEL9-Pool for x86_64" from "Parent Channel"
+    And I enter "Custom channel" as "Channel Summary"
+    And I click on "Create Channel"
+    Then I should see a "Channel Custom Channel for Rocky 9 DVD created" text
+
+  Scenario: Add the Rocky 9 Appstream DVD repository
+    When I follow the left menu "Software > Manage > Repositories"
+    And I follow "Create Repository"
+    And I enter "rocky-9-iso-appstream" as "label"
+    And I enter "http://127.0.0.1/rocky-9-iso/AppStream" as "url"
+    And I uncheck "metadataSigned"
+    And I click on "Create Repository"
+    Then I should see a "Repository created successfully" text
+
+  Scenario: Add the Rocky 9 BaseOS DVD repository
+    When I follow the left menu "Software > Manage > Repositories"
+    And I follow "Create Repository"
+    And I enter "rocky-9-iso-baseos" as "label"
+    And I enter "http://127.0.0.1/rocky-9-iso/BaseOS" as "url"
+    And I uncheck "metadataSigned"
+    And I click on "Create Repository"
+    Then I should see a "Repository created successfully" text
+
+  Scenario: Add both repositories to the custom channel for Rocky 9 DVD
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Custom Channel for Rocky 9 DVD"
+    And I follow "Repositories" in the content area
+    And I select the "rocky-9-iso-appstream" repo
+    And I select the "rocky-9-iso-baseos" repo
+    And I click on "Save Repositories"
+    Then I should see a "repository information was successfully updated" text
+
+  Scenario: Synchronize the repositories in the custom channel for Rocky 9 DVD
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Custom Channel for Rocky 9 DVD"
+    And I follow "Repositories" in the content area
+    And I follow "Sync"
+    And I click on "Sync Now"
+    Then I should see a "Repository sync scheduled" text
+
+  Scenario: The custom channel for Rocky 9 has been synced
+    When I wait until the channel "rocky-9-iso" has been synced
+
+  Scenario: Create CLM filters to remove AppStream metadata
+    Given I am authorized for the "Admin" section
+    When I follow the left menu "Content Lifecycle > Filters"
+    And I click on "Create Filter"
+    And I wait at most 10 seconds until I see modal containing "Create a new filter" text
+    Then I should see a "Create a new filter" text
+    And I enter "ruby-2.7" as "filter_name"
+    And I select "Module (Stream)" from "type"
+    And I enter "ruby" as "moduleName"
+    And I enter "2.7" as "moduleStream"
+    And I click on "Save" in "Create a new filter" modal
+    Then I should see a "ruby-2.7" text
+    When I click on "Create Filter"
+    Then I wait at most 10 seconds until I see modal containing "Create a new filter" text
+    Then I should see a "Create a new filter" text
+    When I enter "python-3.6" as "filter_name"
+    And I select "Module (Stream)" from "type"
+    And I enter "python36" as "moduleName"
```

Comment:
Did you confirmed those are the versions of Ruby and Python that we want to filter for RH-like 9?

Reply:
No, but how can I see the contents of the appstream channels without doing a BV deployment ?

My idea was to test it now so I could fix it before I start 4.3.3 BV.

Reply:
After long discussion in slack, added a comment as why 3.6 makes sense

We don't know yet the "correct" value for SLES 9, it's possible we need a new deployment to determine the correct value. Alternatively, the doc team might write it down.